Modules/Forum/classes/class.ilObjForumAdministrationGUI.php

```diff
@@ -161,18 +159,6 @@ protected function getSettingsForm(): ilPropertyFormGUI
         $form->setFormAction($this->ctrl->getFormAction($this, 'saveSettings'));
         $form->setTitle($this->lng->txt('settings'));
 
-        $frm_radio = new ilRadioGroupInputGUI($this->lng->txt('frm_displayed_infos'), 'forum_overview');
```

Comment:
We should also remove the language variables from the DE and EN language files, if not used anywhere else.